chore(a4a): add README.md with Getting Started instructions (#93747)
Automattic/wp-calypso: client/a8c-for-agencies/README.md

> # A8C for Agencies

Reviewer: ```suggestion
# Automattic for Agencies (A4A)
```

## Getting Started

> You can try out the user-side of Calypso on https://agencies.automattic.com/ or install and run it locally:

Reviewer 2: This will currently cause a `no-literal-urls` ESLint error. Let's fix it by making it a proper link:

```suggestion
You can try out the user-side of Calypso on [https://agencies.automattic.com/](https://agencies.automattic.com/) or install and run it locally:
```

> You can try out the user-side of Calypso on https://agencies.automattic.com/ or install and run it locally:

Reviewer: ```suggestion
You can try out the front-end of A4A on https://agencies.automattic.com/ or install and run it locally:
```
"user-side" sounds like a clunky way of saying "front-end" and "Calypso" should probably be replaced by "A4A"
